cg/cli/upload/clinical_delivery.py

```diff
@@ -30,31 +29,22 @@ def upload_clinical_delivery(context: click.Context, case_id: str, dry_run: bool
 
     click.echo(click.style("----------------- Clinical-delivery -----------------"))
 
-    case: Case = context.obj.status_db.get_case_by_internal_id(internal_id=case_id)
+    case: Case = context.obj.status_db.get_case_by_internal_id(case_id)
     delivery_types: set[str] = case.get_delivery_arguments()
     is_sample_delivery: bool
     is_case_delivery: bool
     is_complete_delivery: bool
     job_id: int
-    is_sample_delivery, is_case_delivery = DeliverAPI.get_delivery_scope(
-        delivery_arguments=delivery_types
-    )
+    is_sample_delivery, is_case_delivery = get_delivery_scope(delivery_types)
```

Comment:
Why do we distinguish a sample delivery and a case delivery? Are not samples always in a case and do we not always deliver the files for all samples in a case?